src/test/chai.t.sol

```diff
@@ -152,6 +152,50 @@ contract ChaiTest is DSTest, ChaiSetup {
         chai.exit(address(this), chaiBal);
         assertEq(dai.balanceOf(address(this)), 100 ether + 0.5 ether - 1);
     }
+    function test_move() public {
+        pot.file("dsr", uint(1000000564701133626865910626));  // 5% / day
+        chai.join(address(this), 10 ether);
+        assertEq(chai.balanceOf(address(this)), 10 ether);
+        assertEq(chai.dai(address(this)), 10 ether);
+        hevm.warp(now + 1 days);
+        assertEq(chai.balanceOf(address(this)), 10 ether);
+        assertEq(chai.dai(address(this)), 10 ether + 0.5 ether);
+
+        chai.move(address(this), address(0xcafebabe), 0.5 ether);
+
+        // we sent one more than requested because of rounding
+        assertEq(chai.dai(address(this)), 10 ether - 1);
+        // cafebabe got no less than requested
+        assertEq(chai.dai(address(0xcafebabe)), 0.5 ether);
+
+        chai.exit(address(this), chai.balanceOf(address(this)));
+        assertEq(dai.balanceOf(address(this)), 100 ether - 1);
+    }
+    function test_move_receive_extra() public {
+        pot.file("dsr", uint(1000000564701133626865910626));  // 5% / day
+        // make chi nasty
+        hevm.warp(now + 100 days);
+        chai.join(address(this), 10 ether);
+        uint chaiBal = rdiv(10 ether, pot.chi());
+        assertEq(chai.balanceOf(address(this)), chaiBal);
+        // 18 wei lost to rounding
```

Comment:
Calling the difference `lost` makes it seem like it's irretrievable, when it's really just interest that is yet too little to make up a smallest denomination of `dai`. I don't have a better word choice, but I think it's important to point out that these rounding errors don't accumulate over time.

Reply:
Agreed, will think about a better way of phrasing these comments.